plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -1205,12 +1208,12 @@ public static Button createPushButton(@NotNull Composite parent, @Nullable Strin
 
     @NotNull
     public static Button createDialogButton(@NotNull Composite parent, @Nullable String label, @Nullable SelectionListener selectionListener) {
-        return createDialogButton(parent, label, null, null, selectionListener);
+        return createDialogButton(parent, label, null, (DBPImage) null, selectionListener);
     }
 
     @NotNull
     public static Button createDialogButton(@NotNull Composite parent, @Nullable String label, @Nullable DBPImage icon, @Nullable String toolTip, @Nullable SelectionListener selectionListener) {
-        return createDialogButton(parent, label, toolTip, icon, GridData.HORIZONTAL_ALIGN_FILL, selectionListener);
+        return createDialogButton(parent, label, toolTip, icon, selectionListener);
```

Comment:
Are we sure that we want to change it in the final sprint?

Reply:
Yes, why? It's just an overloaded method